zerver/openapi/openapi.py

```diff
@@ -13,21 +14,63 @@
 EXCLUDE_PROPERTIES = {
     '/register': {
         'post': {
-            '200': ['max_message_id', 'realm_emoji'],
+            '200': ['max_message_id', 'realm_emoji', 'pointer'],
         },
     },
+    '/settings/notifications': {
+        'patch': {
+            # Extraneous key
+            '200': ['notification_sound', 'enable_login_emails',
+                    'enable_stream_desktop_notifications', 'wildcard_mentions_notify',
+                    'pm_content_in_desktop_notifications', 'desktop_icon_count_display',
+                    'realm_name_in_notifications', 'presence_enabled'],
+        },
+    },
+
     '/zulip-outgoing-webhook': {
         'post': {
             '200': ['result', 'msg', 'message'],
         },
     },
+    '/users/me': {
+        'get': {
+            # Extraneous key
+            '200': ['delivery_email'],
+        },
+    },
+    '/users/{user_id}': {
+        'get': {
+            # Extraneous key
+            '200': ['delivery_email'],
+        },
+        'delete': {
+            # Extraneous key
+            '200': ['delivery_email'],
+        }
+    },
+    '/fetch_api_key': {
+        'post': {
+            # Required key not present in response
+            '200': ['email'],
+        }
+    },
+    '/messages': {
+        'post': {
+            '200': ['deliver_at'],
+        }
+    }
 }
-
+# If we want to exclude the endpoint entirely we just write the
+# endpoint in EXCLUDE_ENDPOINTS else we can write ENDPOINT:METHOD
+# if we just want to skip a single method of that endpoint
+EXCLUDE_ENDPOINTS = ["/users/me/presence", "/messages/matches_narrow",
+                     "/realm/emoji/{emoji_name}:delete"]
```

Comment:
Let's do these 1-per-line with comments above each on why they're here.

Reply:
After the new algo there is no collision so EXCLUDE_ENDPOINT has only one entry `"/realm/emoji/{emoji_name}:delete"` as we have documented this endpoint but not for this method. Done.